board/tests.py

```diff
@@ -1,9 +1,11 @@
+from django.http.response import JsonResponse
+from .models import Board, Image, Post
```

Comment:
pretty sure Python doc import order should be
- builtin
- imported libraries
- local files